### DISCUSSION TITLE
custom ignored directory

### DISCUSSION THREAD

**Discussion by BanDroid** [vikejs/vike#992] [Discussion] [answered]
by default vite-plugin-ssr ignores src, pages, and index folder name. how could i customize this? for example i want a folder named `client` to be ignored too

**Comment by brillout** [comment]
> how could i customize this?

You currently can't. 

> i want a folder named `client` to be ignored too

Why?

**Reply by BanDroid** [reply]
just curious if there's a way to customize it, cause my project structure looks like this and need to separate client and server folder, but since i cant do that, i will stick with src instead of client.

- Root/
  - server/
  - client/
    - pages/
    - renderer/
  - package.json

**Reply by brillout** [reply, marked as answer]
I see, makes sense. You can achieve that with `filesystemRoot`.
